Sources/SBTUITestTunnelServer/SBTUITestTunnelServer.m

```diff
@@ -888,21 +888,48 @@ - (NSDictionary *)commandScrollScrollViewWithIdentifier:(NSString *)elementIdent
                 BOOL expectedIdentifier = [view.accessibilityIdentifier isEqualToString:elementIdentifier] || [view.accessibilityLabel isEqualToString:elementIdentifier];
                 if (expectedIdentifier) {
                     UIScrollView *scrollView = (UIScrollView *)view;
-                                        
+
+                    // Try to understand if the scrollView need to scroll vertically or horizontally (if both are valid we cannot continue)
+                    BOOL canScrollVertically = false;
+                    BOOL canScrollHorizontally = false;
```

Comment:
canScroll -> shouldScroll

Did you consider using a single boolean assuming vertical as the default scroll direction, avoiding the early exit logic at 903-906?

Reply:
Actually my main idea was to let the developer to decide in which direction to scroll adding an enum parameter `scrollDirection`:

```
typedef enum: NSInteger {
    SBTUITestTunnelScrollDirectionHorizontal,
    SBTUITestTunnelScrollDirectionVertical,
} SBTUITestTunnelScrollDirection;
```

to the following methods:

```
- (BOOL)scrollCollectionViewWithIdentifier:(nonnull NSString *)identifier toRow:(NSInteger)row animated:(BOOL)flag
- (BOOL)scrollCollectionViewWithIdentifier:(nonnull NSString *)identifier toElementWithIdentifier:(nonnull NSString 
- (BOOL)scrollScrollViewWithIdentifier:(nonnull NSString *)identifier toElementWithIdentifier:(NSString *)targetIdentifier animated:(BOOL)flag
```

Reply:
I force-pushed a more stable solution without early exit. In this implementation I take the assumption that the vertical scroll is the most common one, and fallback on that in case it's not possible to clearly understand what is the correct scroll direction for the selected scrollView.